plans/chew-datasets/utils/testconfig.go

```diff
@@ -26,59 +26,58 @@ func GetTestConfig(runenv *runtime.RunEnv, acceptFiles bool, acceptDirs bool) (c
 
 	if acceptFiles {
 		// Usage: --test-param file-sizes='["10GB"]'
-		sizes := runenv.StringArrayParamD("file-sizes", []string{"1MB", "10MB", "100MB" /* "1GB", "10GB" */})
-
-		for _, size := range sizes {
-			n, err := humanize.ParseBytes(size)
-			if err != nil {
-				return cfg, err
-			}
-
-			file, err := CreateRandomFile(runenv, os.TempDir(), int64(n))
-			if err != nil {
-				return nil, err
+		sizes, ok := runenv.StringArrayParam("file-sizes")
+		if ok {
+			for _, size := range sizes {
+				n, err := humanize.ParseBytes(size)
+				if err != nil {
+					return cfg, err
+				}
+
+				file, err := CreateRandomFile(runenv, os.TempDir(), int64(n))
```

Comment:
Move the random file creation function to `sdk/runtime`? I think `smlbench` also creates random files, so we should refactor both to use a common util function.